source/MRViewer/MRRibbonNotification.cpp

```diff
@@ -132,7 +137,16 @@ void RibbonNotifier::drawNotifications( float scaling )
         ImGui::SetCursorPosX( 40.0f * scaling );
         if ( notification.onButtonClick )
             ImGui::SetCursorPosY( ImGui::GetCursorPosY() + style.FramePadding.y * 0.5f );
-        ImGui::TextWrapped( "%s", notification.text.c_str() );
+        if ( notification.type != NotificationType::Time )
+            ImGui::TextWrapped( "%s", notification.text.c_str() );
+        else
+        {
+            ImGui::TextWrapped( "%s", notification.header.c_str() );
+            ImGui::SameLine();
+            ImGui::PushStyleColor( ImGuiCol_Text, ImGui::GetColorU32( ImGuiCol_Text, 0.7f ) );
+            ImGui::TextWrapped( "%s", notification.text.c_str() );
+            ImGui::PopStyleColor();
+        }
```

Comment:
why change this?

Reply:
reworked (but time information style will chenge)